test/units/modules/network/nxos/fixtures/nxos_static_route.cfg

```diff
@@ -1,3 +1,3 @@
 ip route 10.10.30.0/24 1.2.4.8
 vrf context test
-  ip route 10.8.0.0/14 15.16.17.18
+ip route 10.8.0.0/14 15.16.17.18
```

Comment:
How do the new changes differentiate between routes under the default and non-default vrf?

Reply:
I found a flaw in my implementation and need to rework this a bit. Let's discuss offline. Thanks.